### ISSUE TITLE
SL not triggered due to error

### ISSUE THREAD

**Issue by offtopic** [MarcelBeining/EazeBot#16]
Hi @MarcelBeining!

Unfortunately this is a major issue -> SL was not triggered:

![image](https://user-images.githubusercontent.com/3337241/89756183-118ae200-dae2-11ea-93ad-7f7526523cd2.png)
I searched my computer for `telegramEazeBot.log` but don't find one -> where are the logs stored for docker version?

I hope the screenshot is enough to fix the issue...

Docker Version: **v2.6.0** (the latest docker version is on **2.6.0 and the standard version is on 2.6.1**)
OS: Windows 10 64 bit

`python -m pip list` executing in the host is IMHO not relevant for the docker version, right?
Version of ccxt: 1.32.12

Best regards, off

**Comment by offtopic** [comment]
Hi Marcel,

i think i fixed the error ... have not the infrastructure at the moment to create a PR:

```
            if self.th.exchange.has['createMarketOrder']:
                params = {}
                params['i_ts'] = self.get_uid()

                if self.th.exchange.name == 'Kraken':
                    params['trading_agreement'] = 'agree'

                try:
                    response = self.safe_run(
                        lambda: self.th.exchange.createMarketSellOrder(self.symbol, self.coinsAvail, params))
```

I have integrated the i_ts into params, instead of additional parameter...

**Comment by MarcelBeining** [comment]
Fixed in [v2.6.2](https://github.com/MarcelBeining/EazeBot/releases/tag/v2.6.2)

Thanks for your patience

**Comment by offtopic** [comment]
Ah, i see my proposed fix was wrong (tmi), thx for the fix!
```i_ts``` is a param from ```self.safe_run``` (facepalm)
